src/routes/auth.route.js

```diff
@@ -38,7 +40,35 @@ const authRouter = Router();
  *       5XX:
  *        description: Unexpected error.
  */
-authRouter.post('/signup', validateNewUser, signup);
+authRouter.post('/signup', validateReqBody(signupSchema), signup);
+
+/**
+ * @swagger
+ *
+ * /api/v1/users:
```

Comment:
Nice documentation provided. I observed that you used `/api/v1/users` which already exist for another route. why not change it to something similar to `/api/v1/users/confirmation/:token` so that this documentation will show up and can also be tested for.